xbmc/video/VideoDatabase.cpp

```diff
@@ -4905,11 +4909,25 @@ void CVideoDatabase::UpdateTables(int iVersion)
     m_pDS->exec("ALTER TABLE settings ADD VideoStream integer");
     m_pDS->exec("ALTER TABLE streamdetails ADD strVideoLanguage text");
   }
+
+  if (iVersion < 104)
+  {
+    m_pDS->exec("ALTER TABLE tvshow ADD duration INTEGER");
+
+    std::string sql = PrepareSQL("SELECT idShow, MAX(c%02d) FROM episode GROUP BY idShow", VIDEODB_ID_EPISODE_RUNTIME);
```

Comment:
Ok longer but this is way more. Every tvshow that have a 2 in 1 episode will have twice the duration, every episode that have a season/series finale longer will get a way longer runtime
Try this in your database:
`select tvshow.c00, MAX(episode.c09)/60 from tvshow JOIN episode ON episode.idShow = tvshow.idShow GROUP BY tvshow.idShow`